### PR TITLE
Restores Previous and Next in Execution page #5102

### LINKED ISSUE #5102: missing "previous" and "next" execution buttons in rd3rc2
**Describe the bug**
Missing "previous" and "next" execution buttons in rd3rc2

**My Rundeck detail**

* Rundeck version: 3.1. rc2
* install type: any
* OS Name/version: ubuntu 18.04
* DB Type/version: any

**To Reproduce**
1. Go to "Activity"
2. Click on an execution
3. You won't see those buttons

**Screenshots**
rd3.0.24, see buttons in the upper right side

Captura de pantalla de 2019-07-30 17-34-11

rd3.1rc2

Captura de pantalla de 2019-07-30 17-36-28

### PR DESCRIPTION
**Pull request by jessemarple**
Resolves #5102 

By popular request we have restored the "Previous" and "Next" links for a job execution. Unlike before, these are not buttons, but instead links in a dropdown.

### CODE REVIEW

File: rundeckapp/grails-app/views/execution/show.gsp [hunk 1]
```diff
@@ -292,7 +315,7 @@ search
                               <g:link controller="scheduledExecution"
                                       action="execute"
                                       id="${scheduledExecution.extid}"
-                                      class=" pull-right"
+                                      class="btn btn-default btn-xs pull-right"
```
Comment (gschueler) [inline rundeckapp/grails-app/views/execution/show.gsp]: revert unrelated change:
```suggestion
                                      class=" pull-right"
```

Comment (jessemarple) [inline rundeckapp/grails-app/views/execution/show.gsp]: This adds a button for the Run Again trigger, in keeping with the button usage on the job page for the "Run Job Now" Button

Comment (jessemarple) [reply rundeckapp/grails-app/views/execution/show.gsp]: This adds a button for the Run Again trigger, in keeping with the button usage on the job page for the "Run Job Now" Button
